### ISSUE TITLE
Get status of the proximity sensor

### ISSUE THREAD

**Issue by WollyWonka** [robertklep/nefit-easy-http-server#52]
Hi, I was wondering if it is possible to retrieve the status of the proximity sensor of the Nefit Easy. In other words, to see if someone is standing in front of the panel?  Alternatively, to see whether the display lights up.
Thanx for considering!
/Wol

**Comment by robertklep** [comment]
I don't think this is possible, the device doesn't expose the sensor value in any way as far as I know.

**Comment by WollyWonka** [comment]
That's a quick response, thank you. Sad to hear this short coming. 
What I actually want to know, if the temperature setpoint is set remote or manually via the thermostat. You might have a suggestion?

**Comment by robertklep** [comment]
I don't have a Nefit device anymore so can't test, but there are a few [endpoints](https://github.com/robertklep/nefit-easy-core/wiki/List-of-endpoints) that you can check to see if they change according to how the setpoint was set:
```
/heatingCircuits/hc1/actualSupplyTemperature
/heatingCircuits/hc1/control
/heatingCircuits/hc1/manualTempOverride/status
/heatingCircuits/hc1/manualTempOverride/temperature
/heatingCircuits/hc1/operationMode
/heatingCircuits/hc1/temperatureAdjustment
/heatingCircuits/hc1/temperatureRoomManual
/heatingCircuits/hc1/type
/heatingCircuits/hc1/usermode
```

**Comment by WollyWonka** [comment]
Great! I'll get into this.
